internal/mode/static/state/graph/gateway_test.go

```diff
@@ -701,16 +719,8 @@ func TestBuildGateway(t *testing.T) {
 							{Kind: "HTTPRoute"},
 						},
 					},
-					"bar-80": {
```

Comment:
Important part: The order of `expected.Listeners` should be the same as [listeners](https://github.com/nginxinc/nginx-gateway-fabric/blob/55b17384cdd4042ffca8dcb8b5c7dec8f6729df9/internal/mode/static/state/graph/gateway_test.go#L640-L649).

Reply:
[3]